### PR TITLE
Angular bank: skeleton

### PR DESCRIPTION
**Pull request by slavidichkov**

### CODE REVIEW

**Comment by slavidichkov** [comment]
@stefan-dimitrov

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,25 @@
+*.class
+out/
+frontend/karma/
+frontend/node_modules/
```
Comment (stefan-dimitrov) [inline .gitignore]: You can have another .gitignore inside `frontend/` directory for these.

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,25 @@
+*.class
+out/
+frontend/karma/
```
Comment (stefan-dimitrov) [inline .gitignore]: `frontend/karma` directory should not be excluded from git. It contains a template file used by your testing configuration.

File: .gitignore (hunk 1 above)
Comment (slavidichkov) [inline .gitignore]: fixed

File: .gitignore (hunk 2 above)
Comment (slavidichkov) [inline .gitignore]: fixed

File: frontend/CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,141 @@
+# 0.3.2 (2014-04-15)
+
```
Comment (mgenov) [inline frontend/CHANGELOG.md]: ChangeLog doesn't look reasonable.

File: frontend/README.md [hunk 4]
```diff
@@ -0,0 +1,380 @@
+# [ngBoilerplate](http://joshdmiller.github.main.java.clouway.com/ng-boilerplate) [![Build Status](https://api.travis-ci.org/ngbp/ngbp.png?branch=v0.3.2-release)](https://travis-ci.org/ngbp/ngbp)
```
Comment (mgenov) [inline frontend/README.md]: Doesn't look relative with the task.

File: frontend/build.config.js [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+     * our reusable components' (`src/common`) template HTML files, while
+     * `atpl` contains the same, but for our app's code. `html` is just our
+     * main HTML file, `less` is our main stylesheet, and `unit` contains our
+     * app's unit tests.
+     */
+    app_files: {
+        js: ['src/**/*.js', '!src/**/*.spec.js', '!src/assets/**/*.js'],
+        jsunit: ['src/**/*.spec.js'],
+
+        coffee: ['src/**/*.coffee', '!src/**/*.spec.coffee'],
```
Comment (mgenov) [inline frontend/build.config.js]: Currently we are not using coffee script, so this configuration should be removed.

File: frontend/package.json [hunk 6]
```diff
@@ -0,0 +1,39 @@
+{
```
Comment (mgenov) [inline frontend/package.json]: Package information looks wrong.

File: frontend/src/app/app.spec.js [hunk 7]
```diff
@@ -0,0 +1,16 @@
+// describe( 'AppCtrl', function() {
```
Comment (mgenov) [inline frontend/src/app/app.spec.js]: It looks like that test configuration of the build script is broken as this test is commented.

File: frontend/tools.md [hunk 8]
```diff
@@ -0,0 +1,223 @@
+# The Tools Used in `ngBoilerplate`
+
```
Comment (mgenov) [inline frontend/tools.md]: Looks irrelevant with the task.

File: pom.xml [hunk 9]
```diff
@@ -0,0 +1,165 @@
...
+      <groupId>com.google.code.gson</groupId>
+      <artifactId>gson</artifactId>
+      <version>2.6.2</version>
+    </dependency>
+  </dependencies>
+
+  <build>
+
+    <plugins>
+      <!--<plugin>-->
```
Comment (mgenov) [inline pom.xml]: All this commented stuff need to be removed.

**Comment by mgenov** [comment]
5000 rows? Is anyone is going to review this stuff ?

File: pom.xml (hunk 9 above)
Comment (slavidichkov) [inline pom.xml]: fixed

File: frontend/tools.md (hunk 8 above)
Comment (slavidichkov) [inline frontend/tools.md]: fixed

File: frontend/src/app/app.spec.js (hunk 7 above)
Comment (slavidichkov) [inline frontend/src/app/app.spec.js]: fixed

File: frontend/package.json (hunk 6 above)
Comment (slavidichkov) [inline frontend/package.json]: fixed

File: frontend/README.md (hunk 4 above)
Comment (slavidichkov) [inline frontend/README.md]: fixed

File: frontend/CHANGELOG.md (hunk 3 above)
Comment (slavidichkov) [inline frontend/CHANGELOG.md]: fixed

**Comment by slavidichkov** [comment]
I added bank from servlets, thats why my PR is so long.
